indico/modules/events/registration/templates/display/registration_summary.html

```diff
@@ -12,7 +12,7 @@
                    {% trans %}Your registration has been completed{% endtrans %}
                </div>
                {% trans -%}
-                   Everything is done.
+                   The process is over.
```

Comment:
IMO this message sounds a bit strange (but so does the old message).

Unless it looks odd maybe we can just omit the message here?